recipe/bld.bat

```diff
@@ -0,0 +1,16 @@
+
+call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\vcvars64.bat"
```

Comment:
`conda-build` should do this for you. So we should be able to drop it.

Reply:
OK, let's try.